src/util/Util.js

```diff
@@ -79,7 +79,7 @@ class Util {
   static escapeMarkdown(text, onlyCodeBlock = false, onlyInlineCode = false) {
```

Comment:
Perhaps add a parameter `onlySpoiler` for this so it's consistent with the other two?

Reply:
I imagined there was a reason not to, due to there not being `onlyStrikeout` or `onlyUnderline`, etc

Reply:
In all honesty `onlySpoiler` would be useful.